src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/ActivityTracker.cs

```diff
@@ -588,6 +588,25 @@ private void ActivityChanging(AsyncLocalValueChangedArgs<ActivityInfo?> args)
             // currently we do nothing, as that seems better than setting to Guid.Emtpy.
         }
 
+        // Assembly bind runtime activity name
+        private const string AssemblyBindName = "AssemblyBind";
```

Comment:
I know we like to call this assembly binding for historic reasons, but our documentation and customers call this assembly loading. E.g.: https://docs.microsoft.com/en-us/dotnet/core/dependency-loading/loading-managed

I would stick with assembly loading for anything that is public facing.

Reply:
Does that mean we don't want to call it assembly binding in the events either? I put them under the `Binder` keyword (that used to be `Fusion`); I didn't want to have them under `Loader`, since I think we'd want to enable them separately. The new events are also under the `AssemblyBinder` task and named `AssemblyBind*`.

Reply:
Would it make sense to call this `AssemblyLoader` / `AssemblyLoad` ?

I would be interested to know what other people (e.g. PMs) think about the right public facing names for this.

Reply:
Or maybe also `AssemblyResolver`/`AssemblyResolve`? Since we have the `AssemblyLoadContext.Resolving` and `AppDomain.AssemblyResolve` events.

Will check with PMs.